Update os_setup.md with python3 documentation

alrojo opened the pull request:
Adds python3 documentation for: "Test the TensorFlow installation"

alrojo committed:
Update os_setup.md with python3 documentation

Adds the python3 documentation for: "Test the TensorFlow installation"

tensorflow-jenkins commented:
Can one of the admins verify this patch?

alrojo commented:
I signed it!

martinwicke commented on tensorflow/g3doc/get_started/os_setup.md:
````diff
@@ -272,6 +272,11 @@ You can find out the directory with the following command:
+```bash
+$ python3 -c 'import os; import inspect; import tensorflow; print(os.path.dirname(inspect.getfile(tensorflow)))'
+```
+
````
Instead of basically repeating all the code with a minor change, could we instead change line 269 and append "(make sure to use the python command you have installed TensorFlow with, for example, use `python3` instead of `python` if you installed for Python 3)"

I think that's probably more readable.

alrojo committed:
Update os_setup.md

Update os_setup.md to @martinwicke comment

alrojo committed:
Merge branch 'master' into patch-1

alrojo commented:
@martinwicke made an update regarding your comment

martinwicke commented on tensorflow/g3doc/get_started/os_setup.md:
```diff
@@ -285,7 +285,7 @@ Extracting data/t10k-images-idx3-ubyte.gz
 ...etc...
 
-# You can alternatively pass the path to the model program file to the python interpreter.
+# You can alternatively pass the path to the model program file to the python interpreter (make sure to use the python distribution you installed TensorFlow to, for example, .../python3.X/... for Python 3).
```
Can you break this comment into several lines? Maybe at 80 characters

martinwicke commented on tensorflow/g3doc/get_started/os_setup.md:
````diff
@@ -266,7 +266,7 @@ The exact location of the Python library depends on your system, but is usually
 ```
 
-You can find out the directory with the following command:
+You can find out the directory with the following command (make sure to use the Python you installed TensorFlow to, for example, use ```python3``` instead of ```python``` if you installed for Python 3):
````
Can you use single backticks instead of triple ones for the inline code? Like this: `python`?

alrojo committed:
Update os_setup.md - line length and backticks

PEP8 line length and single backticks mark for python and python3 according to @martinwicke comments

alrojo committed:
Merge branch 'master' into patch-1

alrojo commented:
@martinwicke made an update regarding your comments (line breaks and backticks)

martinwicke commented:
Great, I'll squash an merge.

martinwicke commented:
Merged. Thanks!
